Makefile

```diff
@@ -85,6 +91,46 @@ EXEC_ARCH := amd64
 DOCKER_ARCH := amd64
 endif
 
+# shellcheck
+SHELLCHECK_VERSION=v0.8.0
+SHELLCHECK_BIN=${TOOLS_DIR}/shellcheck
+SHELLCHECK_ARCHIVE := shellcheck-$(SHELLCHECK_VERSION).$(OS).$(HOST_ARCH).tar.xz
+ifeq (darwin, $(OS))
+ifeq (arm64, $(HOST_ARCH))
+SHELLCHECK_ARCHIVE := shellcheck-$(SHELLCHECK_VERSION).$(OS).x86_64.tar.xz
+endif
+else ifeq (linux, $(OS))
+ifeq (armv7l, $(HOST_ARCH))
+SHELLCHECK_ARCHIVE := shellcheck-$(SHELLCHECK_VERSION).$(OS).armv6hf.tar.xz
+endif
+endif
+
+# golangci-lint
+GOLANGCI_LINT_VERSION=1.51.2
```

Comment:
latest version available at the moment: 1.53.3

Reply:
Done